pizzeria-backend/Program.cs

```diff
@@ -18,7 +18,8 @@
 
 // Add services to the container.
 
-builder.Services.AddControllers();
+builder.Services.AddControllers().AddNewtonsoftJson(options =>
+    options.SerializerSettings.ReferenceLoopHandling = Newtonsoft.Json.ReferenceLoopHandling.Ignore);
```

Comment:
I saw this on the other MR and I was wondering - why do you need this?

Reply:
Infinite reference loops were happening when using .Include to load the needed relationships